### ISSUE TITLE
Encourage the library authors to publish an Ivy distribution.

### ISSUE THREAD

**Issue by Odonno** [danielsogl/awesome-cordova-plugins#3976]
Like it is said, will you consider moving to Ivy?

We are currently having this kind of message:

```
Processing legacy "View Engine" libraries:

- @ionic-native/core [module/esm5] (https://github.com/ionic-team/ionic-native.git)
- @ionic-native/in-app-browser [module/esm5] (https://github.com/ionic-team/ionic-native.git)

Encourage the library authors to publish an Ivy distribution.
```

**Comment by jpike88** [comment]
@danielsogl would appreciate an answer to this thanks

**Comment by QuentinFarizon** [comment]
.

**Comment by ghenry22** [comment]
any thoughts on this from anyone?

**Comment by QuentinFarizon** [comment]
.

**Comment by HarelM** [comment]
Would be nice to have this warning removed...

**Comment by HarelM** [comment]
Bump

**Comment by HarelM** [comment]
Closed via #4369 I believe...
Thanks @jkorrek & @ghenry22!!
